packages/mui-base/src/SelectUnstyled/useSelect.ts

```diff
@@ -0,0 +1,282 @@
+import * as React from 'react';
```

Comment:
Should we cover these in tests somehow? I am asking because it is not clear what strategy we use for testing (whether we test the component as end to end test, or the hooks, or the reducers).

Reply:
The SelectUnstyled tests cover useSelect as well. As for reducer tests, I included some in useListbox as it helped me during development. They cover more basic functionality of the listbox.